src/spoke/libraries/ReserveFlagsMap.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: UNLICENSED
+// Copyright (c) 2025 Aave Labs
+pragma solidity ^0.8.20;
+
+type ReserveFlags is uint8;
+
+using ReserveFlagsMap for ReserveFlags global;
```

Comment:
is there another way to do this, more aligned with how we use other of libraries? thinking that `using global` is a one-off in the code

Reply:
It prevents from importing both the type and the lib in every file, and repeating the `using` everywhere.
We currently don't have convention for those defined types, so if we want we can adapt.

Reply:
It prevents from importing both the type and the lib in every file, and repeating the `using` everywhere.
We currently don't have convention for those defined types, so if we want we can adapt.